analytics/config/config.go

```diff
@@ -7,48 +7,53 @@ import (
 	"github.com/prebid/prebid-server/config"
 )
 
+type analyticsModule analytics.PBSAnalyticsModule
```

Comment:
Any reason why we need this new type to represent the interface? `analyticsModule` is serving pretty much as an alias to `analytics.PBSAnalyticsModule`

Reply:
I'm using it for the test. Not sure, it's the best approach to check if the module is loaded.
A better solution will be a dedicated type alias for each module in order to check their presence in the enabledAnalytics slice.